core/src/com/metalrain/stocksimulator/state/GameState.java

```diff
@@ -316,5 +324,13 @@ private SerializedEntityList toSerializedEntityArray(ImmutableArray<Entity> enti
         for (Entity e:entities) l.list.add(new SerializedEntity(e));
         return l;
     }
+    private String getRandomStockName() {
+        if (random.nextBoolean()) {
```

Comment:
Don't choose 3vs4 using a boolean, do something like this

int length = random.nextInt(2)+2; (2,3,4 valid lengths)
string output = "";
for (int i=0;i<length;i++) output += randomLetter();